src/main/java/org/openstreetmap/atlas/geography/atlas/raw/sectioning/WaySectionProcessor.java

```diff
@@ -462,8 +494,13 @@ private Atlas cutSubAtlasForOriginalShard(final Atlas atlas)
      */
     private void distinguishPointsFromShapePoints(final WaySectionChangeSet changeSet)
     {
+        final Time time = Time.now();
+        logger.info("Started Shape Point Detection...");
```

Comment:
Mention which shard here? Is it possible given the only parameter passed in is the changeset?

Reply:
Good point, it is, we have all loaded shards!